src/device/pf_cmina.c

```diff
@@ -1341,13 +1342,13 @@ void pf_cmina_show (pnet_t * net)
       net->cmina_current_dcp_ase.station_name);
    printf ("\n");
 
-   printf ("Default device_vendor          : <%s>\n", p_cfg->device_vendor);
+   printf ("Default device_vendor          : <%s>\n", p_cfg->product_name);
```

Comment:
"Default product_name ..." also in two following printf lines.

Reply:
done